lib/view_component/system_test_helpers.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module ViewComponent
+  module SystemTestHelpers
+    include TestHelpers
+    include Capybara::DSL
+
+    def page
+      Capybara.current_session
+    end
+
+    def with_rendered_component_in_browser(component, **options, &block)
+      layout = options[:layout] || false
+
+      opts = {
+        layout: layout,
+        locals: {
+          render_args: {
+            component: component,
+            template: "view_components/preview",
+            hide_preview_source: true
+          }
+        }
+      }
+      html = controller.render_to_string("view_components/preview", opts)
+
+      # Add './tmp/view_component_integrations/' directory if it doesn't exist to store the rendered component html
+      FileUtils.mkdir_p("./tmp/view_component_integrations/") unless Dir.exist?("./tmp/view_component_integrations/")
```

Comment:
I'm curious, why create this folder? Why not just write a `Tempfile` and not worry about specifying the path?

Reply:
Yeh it doesn't really matter anymore since the files will go away anyways. `/tmp` by itself should be sufficient :)